docs/data-collection/tear-down.md

```diff
@@ -28,8 +28,16 @@
 - [ ] Give the participant the corresponding compensation for the participation and transportation.
 - [ ] Request a signed receipt of the amount of the financial compensation.
 
+!!! important "To boost efficiency, two people can work simultaneously on the following steps — one inside the scanning room and the other outside. They can follow the designated sections [inside the scanning room](#after-scan-inside-the-scanner-room) and [outside the scanning room](#after-scan-outside-the-scanner-room) in parallel."
+
 ## AFTER SCAN, inside the scanner room
 
+- [ ] Unplug the two cables connected to the ET (signal and power). Put those extremities aside far from the scanner.
+- [ ] Disconnect the GA and RB tubes.
+
+- [ ] **With someone outside in the control room**:
+    - [ ] Carefully extract the four cables one by one (the fiber and power of the ET, the tube of RB and the tube of the GA) back through the access tube. 
+  
```

Comment:
This needs more coordination (like for the communication to check the tightness of the RB)
```suggestion
- [ ] Agree with another experimenter, who will exit the room, sit on the other side of the access tubes in the Control Room, and pull cables (fiber and power of the ET) and tubes (RB and GA) out of the room.
    At the same time, you carefully feed them so they do not suffer abrasion from touching the edges of the tube.
```